CHANGELOG.md

```diff
@@ -2,25 +2,36 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.12.1] - 2024-09-10
+
+### Changed
+
+- Better performances for the following methods:
```

Comment:
Nit: There is no performance gain in this SDK. You have just updated the endpoints to v2.

Reply:
However the user does not see it, it does 0 difference for it, should I write it down instead ?